ecosystem/typescript/sdk/src/client/core.ts

```diff
@@ -26,12 +26,14 @@ async function axiosRequest<Request, Response>(
   url: string,
   method: "GET" | "POST",
   body?: Request,
+  mediaType?: string,
```

Comment:
Let's call it contentType instead, since that's the actual header we're using.

Reply:
done